### PR TITLE
Content requests

### PR DESCRIPTION
**Pull request by Threore**
Changes the text in various sections. Solves issues #41, #42, #45, #46, #49, #50, #51 

Issue #59 is obsoleted by using title and subtitle as they should be used.

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 export default {
   hero: {
-    title: 'Outsite LGBT+ society',
-    subtitle: "Delft's society for queer youth",
+    title: 'Outsite',
```
Comment (casperboone) [inline lang/en.js]: When the title is just 'Outsite', I think it becomes redundant. Maybe we can do something along the lines of:

**Delft's LGBT+ Youth Association**
Where you can be yourself without judgement

(I think this subtitle is a bit cringy, but something like this)

File: lang/nl.js [hunk 2]
```diff
@@ -48,10 +49,11 @@ export default {
         vereniging kan vertellen en je kan voorstellen aan wat mensen.`
     },
     bar_buddy: {
-      title: 'Barbuddy Systeem',
-      description: `Wil jij een keertje langskomen op een baravond maar ben je bang dat je je niet meteen op je gemak
-        voelt? Meld je dan aan voor een barbuddy! Een barbuddy is iemand die jou de ins en outs kan vertellen over de 
-        vereniging, maar ook iemand die jou voor kan stellen aan nieuwe mensen.`,
+      title: `Barbuddy's`,
+      description: `Wil jij een keertje langskomen op een baravond maar ben je nog niet bekend met de vereniging? Meld
+      je dan aan voor een barbuddy! Een barbuddy is iemand die van tevoren met je afspreekt zodat je samen naar de
```
Comment (casperboone) [inline lang/nl.js]: Can we keep the indentation here? :)

File: lang/en.js (hunk 1 above)
Comment (Threore) [reply lang/en.js]: Fixed, but with the tagline: "for everyone up to 28 years"

File: lang/nl.js (hunk 2 above)
Comment (Threore) [reply lang/nl.js]: Sure, but there are many other places where this indentation is inconsistent. Fixing all of them I consider out of scope.

**Pull request merged**